app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/NewTransferNavHost.kt

```diff
@@ -52,5 +54,17 @@ fun NewTransferNavHost(navController: NavHostController, closeActivity: () -> Un
             // TODO: Use correct TransferType instead of hard-coded value
             UploadSuccessScreen(transferType = TransferType.MAIL)
         }
+        composable<FilesDetailsDestination> {
+            val filesDetailsDestination: FilesDetailsDestination = it.toRoute()
+            FilesDetailsScreen(
+                navigateToDetails = { fileId ->
+                    navController.navigate(FilesDetailsDestination(fileId))
+                },
+                fileId = filesDetailsDestination.fileId,
+                withSpaceLeft = false, onCloseClicked = {},
```

Comment:
Can't we already code `onCloseClicked`?

Reply:
Yes, I think so too, definitely

Reply:
As we discussed with @sirambd and you, we'll do nothing with the cross as it adds a lot of complexity to the app to remove all the stack of `FileDetailsScreen` depending of where we came from.